src/components/Checkout/AutoDepositCTA.vue

```diff
@@ -0,0 +1,323 @@
+				<div class="tw-text-center tw-relative">
+					<div class="payment-dropin-invalid-cover" v-if="$v.$invalid"></div>
+					<auto-deposit-drop-in-payment-wrapper
+						class="tw-mx-auto"
+						:amount="adAmount"
+						:donate-amount="donation"
+						:day-of-month="dayOfMonth"
+						@complete-transaction="completeADBraintree"
+					/>
+				</div>
+			</div>
+		</form>
+	</div>
+</template>
+
+<script>
+import numeral from 'numeral';
+import { validationMixin } from 'vuelidate';
+import { required, minValue, maxValue } from 'vuelidate/lib/validators';
+import { mdiClose } from '@mdi/js';
+
+import AutoDepositDropInPaymentWrapper from '@/components/AutoDeposit/AutoDepositDropInPaymentWrapper';
+import KvCurrencyInput from '@/components/Kv/KvCurrencyInput';
+
+import KvMaterialIcon from '~/@kiva/kv-components/vue/KvMaterialIcon';
+import KvSelect from '~/@kiva/kv-components/vue/KvSelect';
+import KvTextInput from '~/@kiva/kv-components/vue/KvTextInput';
+
+const imageRequire = require.context('@/assets/images/kiva-classic-illustrations/', true);
+
+// arbitrary number below 10k
+const maxAmount = 10000;
+
+export default {
+	components: {
+		AutoDepositDropInPaymentWrapper,
+		KvCurrencyInput,
+		KvMaterialIcon,
+		KvSelect,
+		KvTextInput,
+	},
+	props: {
+		// TODO: Implement Contentful Driven Content
+		headline: {
+			type: String,
+			default: 'Finish your auto-deposit sign up'
+		},
+		// TODO: Implement Contentful Driven Content
+		bodyCopy: {
+			type: String,
+			// eslint-disable-next-line max-len
+			default: ''
+		},
+		// TODO: Implement Contentful Driven Content
+		buttonText: {
+			type: String,
+			default: 'Complete auto-deposit'
+		}
+
+	},
+	mixins: [
+		validationMixin
+	],
+	validations: {
+		adAmount: {
+			required,
+			minValue: minValue(5),
+			maxValue: maxValue(maxAmount),
+		},
+		donation: {
+			minValue: minValue(0),
+			maxValue: maxValue(maxAmount),
+			combinedTotal(value) {
+				return value + this.adAmount < maxAmount && value + this.adAmount > 0;
+			}
+		},
+		dayOfMonth: {
+			required,
+			minValue: minValue(1),
+			maxValue: maxValue(31)
+		},
+	},
+	data() {
+		return {
+			adAmount: 5,
+			adOptionSelected: 5,
+			adAmountOptions: [
+				{
+					value: 35,
+					label: `${numeral(35).format('$0,0.00')} / month`,
+				},
+				{
+					value: 30,
+					label: `${numeral(30).format('$0,0.00')} / month`,
+				},
+				{
+					value: 25,
+					label: `${numeral(25).format('$0,0.00')} / month`,
+				},
+				{
+					value: 20,
+					label: `${numeral(20).format('$0,0.00')} / month`,
+				},
+				{
+					value: 15,
+					label: `${numeral(15).format('$0,0.00')} / month`,
+				},
+				{
+					value: 10,
+					label: `${numeral(10).format('$0,0.00')} / month`,
+				},
+				{
+					value: 5,
+					label: `${numeral(5).format('$0,0.00')} / month`,
+				},
+				{
+					value: 'other',
+					label: 'Other',
+				},
+			],
+			dayOfMonth: parseInt(new Date().getDate(), 10),
```

Comment:
Is this parseInt needed?

Reply:
For some reason it was "becoming" a string and it seemed be at initialization time. Haven't seen the warning since...